Gems/Atom/Utils/Code/Include/Atom/Utils/ImGuiCpuProfiler.h

```diff
@@ -19,31 +23,49 @@ namespace AZ
 
     namespace Render
     {
-        struct ThreadRegionEntry 
+        struct ThreadRegionEntry
         {
             AZStd::thread_id m_threadId;
             AZStd::sys_time_t m_startTick = 0;
             AZStd::sys_time_t m_endTick = 0;
         };
 
+        // Stores data about a region that is agreggated from all collected frames
+        // Data collection can be toggled on and off through m_record. 
+        struct RegionStatistics
+        {
+            float CalcAverageTimeMs() const;
+            void RecordRegion(const AZ::RHI::CachedTimeRegion& region);
+
+            bool m_draw;
+            bool m_record = true;
+            u64 m_invocations;
```

Comment:
This also need initialization. you can set it to 0 but I see that you use this as a denominator below so maybe init to 1.